src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java

```diff
@@ -98,6 +102,17 @@ HumanReadableMeasureInformationModel buildMeasureInfo(Measure measure) {
             measure.getMeasureMetaData() != null
                 ? measure.getMeasureMetaData().getMeasureDefinitions()
                 : null)
+        .references(
+            measure.getMeasureMetaData().getReferences().stream()
```

Comment:
I just checked there might be a chance if `measure.getMeasureMetaData().getReferences()` is null instead of an empty array

Reply:
Updated it